_layouts/docs.html

```diff
@@ -423,6 +445,7 @@
   <script src="/js/highlight.pack.js"></script>
   <script>hljs.initHighlightingOnLoad();</script>
   <script src="/js/alljs.js"></script>
+  <script src="/js/bootstrap.min.js"></script>
```

Comment:
Maybe it is the one Js in the world that we don't have yet. ;) What functions are you using from it?